packages/verkle/src/util/lock.ts

```diff
@@ -0,0 +1,42 @@
+// Based on https://github.com/jsoendermann/semaphore-async-await/blob/master/src/Semaphore.ts
+export class Lock {
```

Comment:
This file also exists in `packages/util/src/lock.ts`?

Reply:
Didn't realized that (had copied from trie), will refactor.

Reply:
Done